### PR TITLE
TRUNK-5035: Use Boolean.valueOf to test strings

### PR DESCRIPTION
**Pull request by PKatGITHUB**
## Related Issue

see https://issues.openmrs.org/browse/TRUNK-5035

## Checklist:

- [x] My pull request only contains one single commit.
- [x] My pull request is based on the latest master branch
  `git pull --rebase upstream master`.
- [x] I ran `mvn clean package` right before creating this pull request and
  added all formatting changes to my commit.
- [x] My code follows the code style of this project.
- [x] I have read the **CONTRIBUTING** document.
- [x] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

File: api/src/main/java/org/openmrs/api/impl/EncounterServiceImpl.java [hunk 1]
```diff
@@ -881,7 +881,7 @@ private boolean userHasEncounterPrivilege(Privilege privilege, User user) {
 	public void checkIfEncounterTypesAreLocked() {
 		String locked = Context.getAdministrationService().getGlobalProperty(
 		    OpenmrsConstants.GLOBAL_PROPERTY_ENCOUNTER_TYPES_LOCKED, "false");
-		if ("true".equals(locked.toLowerCase())) {
```
Comment (dkayiwa) [inline api/src/main/java/org/openmrs/api/impl/EncounterServiceImpl.java]: Can you do like for the original code which had space between the if statement and (?

Comment (PKatGITHUB) [inline api/src/main/java/org/openmrs/api/impl/EncounterServiceImpl.java]: yea sure

Comment (PKatGITHUB) [inline api/src/main/java/org/openmrs/api/impl/EncounterServiceImpl.java]: @dkayiwa do i hv to create a new PR or can i edit the file here only?

Comment (dkayiwa) [inline api/src/main/java/org/openmrs/api/impl/EncounterServiceImpl.java]: Edit this and squash. Did you get a chance to look at https://wiki.openmrs.org/display/docs/Pull+Request+Tips

**Comment by dkayiwa** [comment]
Is the JIRA ticket link on this pull request correct?

File: api/src/main/java/org/openmrs/api/impl/EncounterServiceImpl.java (hunk 1 above)
Comment (PKatGITHUB) [inline api/src/main/java/org/openmrs/api/impl/EncounterServiceImpl.java]: @dkayiwa is it ok now ? :)

**Comment by PKatGITHUB** [comment]
@dkayiwa yes the ticket name is TRUNK-5035.

**Comment by dkayiwa** [comment]
Does the JIRA ticket link on this pull request look correct? 😊

**Comment by dkayiwa** [comment]
Are you looking at the link below "Related Issue"

**Comment by PKatGITHUB** [comment]
@dkayiwa can you please tell me where i am wrong?

**Comment by dkayiwa** [comment]
Can you read from the top of this page and tell me what you see?

**Comment by PKatGITHUB** [comment]
@dkayiwa  updated it.
Thank you very much for your help.
And sorry for the inconvinience.

**Comment by dkayiwa** [comment]
Did you get a chance to look at this? https://wiki.openmrs.org/display/docs/Pull+Request+Tips

**Comment by PKatGITHUB** [comment]
@dkayiwa i read it... and have followed each guideline.
Am i still missing something?

**Pull request merged**